src/detect-engine-analyzer.c

```diff
@@ -915,12 +916,18 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
             }
             case DETECT_SEQ: {
                 const DetectSeqData *cd = (const DetectSeqData *)smd->ctx;
-
                 jb_open_object(js, "seq");
                 jb_set_uint(js, "number", cd->seq);
                 jb_close(js);
                 break;
             }
+            case DETECT_TCPMSS: {
+                const DetectU16Data *cd = (const DetectU16Data *)smd->ctx;
+                jb_open_object(js, "tcp_mss");
+                rs_detect_u16_dump(js, cd);
```

Comment:
not a fan of the `dump` naming, can we just call it `to_json` or `to_jb` or similar?

Reply:
Sure